pf/internal/schemashim/resource_map.go

```diff
@@ -55,3 +55,7 @@ func (m *schemaOnlyResourceMap) Range(each func(key string, value shim.Resource)
 func (*schemaOnlyResourceMap) Set(key string, value shim.Resource) {
 	panic("Set not supported - is it possible to treat this as immutable?")
 }
```

Comment:
This experiment has given me pretty good confidence that we're only using `Set` to add aliases today (Although since it's a public interface, it's possible for random providers to be calling it somewhere. It would be good to remove it in a future major release of the bridge.)